db/models/user.js

```diff
@@ -1,21 +1,48 @@
 import mongoose from "mongoose";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
 
 const schema = new mongoose.Schema(
+  {
+    fullname: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    password: {
+      type: String,
+    },
+  },
+  {
+    timestamps: true,
+  }
+);
+
+/**
+ *
+ * @param {string} password - to check the user password
```

Comment:
The description here is invalid, shouldn't it be, `password entered by the user`?